src/metabase/api/actions.clj

```diff
@@ -36,10 +38,10 @@
 (defn- kwdize-filter [strings-to-kw query]
   (-> query
       (update :type keyword)
-      (update-in [:query :filter]
-                 #(walk/postwalk
-                   (fn [x] (if (and (string? x) (contains? strings-to-kw x)) (keyword x) x))
-                   %))))
+      (m/update-existing-in [:query :filter]
```

Comment:
create action doesn't need a filter, so this would add a nil filter to the query, which is illegal according to the mbql.s/Query schema.